### DISCUSSION TITLE
Velero recovery data prompt error downloading backup

### DISCUSSION THREAD

**Discussion by huiyichanmian** [minio/minio#17019] [Q&A] [answered]
## Expected Behavior
Velero can recover data normally

## Current Behavior
I now use the latest version of minio as the back-end storage of velero1.8. I can successfully back up the data to minio, but during recovery, the restore resource in velero throws an exception: error downloading backup: error copying Backup to temp file：rpc error: code = Unknown desc = error getting object adm/backups/vb-20230412165105/vb-20230412165105.tar.gz: RequestError: send request failed
caused by: Get http://192.168.xx.xx:44088/adm/adm/backups/vb-20230412165105/vb-20230412165105.tar.gz: read tcp 10.244.36.106:60114->192.168.xx.xx:44088: read: connection reset by peer

## Steps to Reproduce (for bugs)
1.Use velero1.8 for backup
2.Failed to recover data using velero. Check restore resources to find the problem of throwing link errors.
3.When I create a bucket using velero, I specify s3ForcePathStyle = "true" and insecureSkipTLSVerify = "true"
4.Recovery is normal when using this version of RELEASE.2022-05-08T23-50-31Z minio, but I will now replace it with the latest one and throw an error

## Your Environment

* Version used (`minio --version`): RELEASE.2023-04-07T05-28-58Z
* Server setup and configuration:
* Operating System and version (`uname -a`): Linux 3.10.0-1062.el7.x86_64 #1 SMP Wed Aug 7 18:08:02 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux

**Comment by klauspost** [comment, marked as answer]
Most likely something in-between Velero and MinIO has a timeout that is hit.

If you don't think so, check your server logs to see if there is anything in there.
